File: official/docs/golang/v2/smartrate/retrieve-estimated-delivery-date.go
package example

import (
	"fmt"
	"os"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
	client := easypost.New(apiKey)

	shipment, _ := client.GetShipment("shp_...")
	estimatedDeliveryDates, _ := client.GetShipmentEstimatedDeliveryDate(shipment.ID, "YYYY-MM-DD")

	fmt.Println(estimatedDeliveryDates)
}

File: official/docs/python/v9/smartrate/retrieve-estimated-delivery-date.py
import easypost

client = easypost.EasyPostClient("EASYPOST_API_KEY")

estimated_delivery_dates = client.shipment.retrieve_estimated_delivery_date(
    "shp_...",
    planned_ship_date="YYYY-MM-DD",
)

print(estimated_delivery_dates)
